bin/apply-template.py

```diff
@@ -0,0 +1,41 @@
+import jinja2
+import json
+import shutil
+import os
+
+def process_template_files(major_version, version, platform):
+    templateLoader = jinja2.FileSystemLoader(searchpath="./templates/")
+    templateEnv = jinja2.Environment(loader=templateLoader)
+
+    template = templateEnv.get_template(f"{platform}.Dockerfile.template")
+    input_parameter = {}
+    input_parameter['CORRETTO_VERSION'] = version
+    input_parameter['MAJOR_VERSION'] = major_version
+    if platform == 'alpine':
+        os_versions = ['3.12', '3.13', '3.14']
```

Comment:
Please add a comment that verify-images has to get updated when this changes.

Reply:
Added.